client/my-sites/checkout/checkout/prepurchase-notices/index.jsx

```diff
@@ -70,11 +70,19 @@ const PrePurchaseNotices = ( { cart } ) => {
 		isJetpackMinimumVersion( state, siteId, BACKUP_MINIMUM_JETPACK_VERSION )
 	);
 
+	// All these notices (and the selectors that drive them)
+	// require a site ID to work. We should *conceptually* always
+	// have a site ID handy; consider this a guard, or an
+	// explicit declaration that all code beyond this point can
+	// safely assume a site ID has been defined.
+	if ( ! siteId ) {
+		return false;
```

Comment:
Good idea, and awesome comment. 😁  However, since this stops rendering the component, we should return `null` instead of `false` (React components can return either a React element or `null`.)

Reply:
I'm not sure how my brain misremembered that fact. In my confusion, I even looked it up to be sure! `null` it is!